website_partner_privacy/templates/portal_template.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="portal_my_details" inherit_id="portal.portal_my_details">
+        <xpath expr="//div[hasclass('col-lg-8')]" position="after">
+            <div class="col-lg-4">
+                <div class="row">
+                    <t t-call="website_partner_privacy.privacy_fields"/>
+                </div>
+            </div>
+        </xpath>
+    </template>
+
+    <template id="privacy_fields">
```

Comment:
Can you use bootstrap classes here instead of specific style?

For the checkboxes you could use https://getbootstrap.com/docs/5.3/forms/checks-radios/#switches that way you don't need `vertical-align: text-bottom;height:18px;width:20%` to make it look nice

![image](https://github.com/user-attachments/assets/4dabb7ae-2f33-4f09-956d-017cfa5327ff)

Reply:
I can help with that if you want to